### PR TITLE
Created benchmark for repository update

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #98 
Added another benchmark for repository update.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 please, could you take a look?

File: src/main/java/com/artipie/debian/benchmarks/RepoUpdateBench.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            new Config.FromYaml(
+                "my-deb",
+                Yaml.createYamlMappingBuilder().add("Architectures", "amd64")
+                    .add("Components", "main").build(),
+                new InMemoryStorage()
+            )
+        );
+        deb.updatePackages(
+            this.debs,
+            new Key.From(String.format("Packages-%s.gz", UUID.randomUUID().toString()))
```
Comment (g4s8) [inline src/main/java/com/artipie/debian/benchmarks/RepoUpdateBench.java:121]: @olenagerasimova `UUID.randomeUUID()` could hang on lack of entropy in kernel, so I'd prepare it ahead of benchmarks, alternatively you could create a method to generate new `UUID` using not secure RNG (just look how `UUID.randomUUID()` is implemented, and copy-paste it with replacing `SecureRandom` with `Random`)

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, actually, there is no need to random uuid, it's enough to add some counter and add to the file name.

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**